content/faq/bookmarks/android-track-recording/index.es.md

```diff
@@ -0,0 +1,21 @@
+---
+title: ¿Por qué las pistas no se graban de forma fiable en segundo plano en Android?
+description: La configuración predeterminada de optimización de la batería en Samsung, Huawei, Google, Xiaomi, OnePlus, Meizu, Asus, Wiko, Lenovo, Oppo, Vivo, Realme, Sony, Motorola, HTC y otros dispositivos puede detener o cerrar la aplicación Organic Maps en segundo plano.
+updated: "2024-10-05"
+
+taxonomies:
+faq: ["Marcadores y rutas"]
+
+extra:
+order: 30
+---
+
+La configuración predeterminada de optimización de la batería en Samsung, Huawei, Google, Xiaomi, OnePlus, Meizu, Asus, Wiko, Lenovo, Oppo, Vivo, Realme, Sony, Motorola, HTC y otros dispositivos puede detener o cerrar la aplicación Organic Maps en segundo plano.
+
+Esto es especialmente cierto para las versiones modernas de Android:
+- Android 14
+- Android 13
+- Android 12
+- Android 11
+
+Los pasos exactos sobre cómo hacer que Organic Maps (y otras aplicaciones) funcionen en segundo plano se enumeran aquí: https://dontkillmyapp.com/
```

Comment:
I would suggest adding a note indicating that this webpage is in English before the colon: "(en inglés)"